lib/highLevelConsumer.js

```diff
@@ -523,11 +518,13 @@ HighLevelConsumer.prototype.offsetRequest = function (payloads, cb) {
  * @param {Client~failedToRegisterConsumerCallback} cb A function to call the consumer has been registered
  */
 HighLevelConsumer.prototype.registerConsumer = function (cb) {
-    this.client.zk.registerConsumer(this.options.groupId, this.id, this.payloads, function (err) {
+    var zk = this.client.zk;
+    var groupId = this.options.groupId;
+    this.client.zk.registerConsumer(groupId, this.id, this.payloads, function (err) {
```

Comment:
Actually, I just noticed how `zk` is probably a bad var name choice as it conflicts with previous global var `zk = require('./zookeeper')` assignment statement and might be just confusing